README.md

```diff
@@ -196,7 +205,7 @@ if not maccor_interface.start():
 maccor_interface.set_channel_variable(var_num = 1, var_value = 0.01)
 ```
 
-## <a name="Direct"></a>Direct Control
+### Direct Control
 
 Here is an example of how to bypass using a test procedure all together and control the channel directly using direct control.  Note the channel safety limits will be set before the test is started. **WARNING DIRECT CONTROL IS POTENTIALLY DANGEROUS. BE CAREFUL AND MAKE SURE YOU UNDERSTAND EXACTLY WHAT YOUR CODE IS DOING.**
```

Comment:
Maybe format the warning as a quote or something. Ideally, we should setup a different formatting for warning  (orange) vs note (grey) vs error (red) background. But a quote can suffice for now if we want to keep it simple.